core/src/main/scala/stryker4s/mutants/findmutants/FileCollector.scala

```diff
@@ -91,9 +91,9 @@ class FileCollector(implicit config: Config) extends SourceCollector with Loggin
     * Is the file in the target folder, and thus should not be copied over
     */
   private[this] def isTarget(file: File): Boolean = {
-    val relativePathString = file.relativePath.toString
+    val pathString = file.toString
```

Comment:
This is causing some failing tests. The relative path is done because if the project happens to be somewhere in a folder named target, it should still copy the files

Reply:
The problem with that is that using relativePath the target filter doesn't work, so if you run Stryker twice, the second one will try to mutate the mutated files of the first one.

It is easy to replicate. Using relativePath just run `sbt stryker` twice inside `test-1` project. Do not use `scripted`, run SBT inside test-1 directory like if it was a project that use Stryker plugin.

Reply:
Hmm, I think that would be the starting / in the match. I'll see if I can fix it, because I think the relative method is preferrable

Reply:
Oh, I found the issue.

> base dir path
> /home/scelestino/dev/stryker4s2/sbt/src/sbt-test/sbt-stryker4s/test-1
> 
> file path
> /home/scelestino/dev/stryker4s2/sbt/src/sbt-test/sbt-stryker4s/test-1/target/stryker4s-3861747083202168747/src/main/scala/example/Example.scala
> 
> relative
> target/stryker4s-3861747083202168747/src/main/scala/example/Example.scala

Relative doesn't start with slash :confused: 

I'm going to fix isTarget method.